tests/executable/test space.tex

```diff
@@ -0,0 +1,4 @@
+\documentclass{article}
```

Comment:
I must admit, I don't really understand enough about plain TeX to be able to make any intelligent progress on this. It would be nice to use the already-existing format files, but I'm not sure if we can make guarantees about whether they'll be generated already. What do you suggest?